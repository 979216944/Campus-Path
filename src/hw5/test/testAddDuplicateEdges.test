# Tests that adding duplicate edges does not affect and corrupt the graph.

# Create graph
CreateGraph g

# Add nodes to g
AddNode g a
AddNode g b
AddNode g c

# Add Edges to g
AddEdge g a a aa
AddEdge g a b ab
AddEdge g a c ac

# Add Duplicate Edges to g
AddEdge g a a aa
AddEdge g a b ab
AddEdge g a c ac

# List nodes and the outgoing edges
ListNodes g
ListChildren g a
ListChildren g b
ListChildren g c