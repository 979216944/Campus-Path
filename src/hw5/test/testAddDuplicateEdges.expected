# Tests that adding duplicate edges does not affect and corrupt the graph.

# Create graph
created graph g

# Add nodes to g
added node a to g
added node b to g
added node c to g

# Add Edges to g
added edge aa from a to a in g
added edge ab from a to b in g
added edge ac from a to c in g

# Add Duplicate Edges to g
added edge aa from a to a in g
added edge ab from a to b in g
added edge ac from a to c in g

# List nodes and the outgoing edges
g contains: a b c
the children of a in g are: a(aa) b(ab) c(ac)
the children of b in g are:
the children of c in g are: